orca/scripts/orca.sh

```diff
@@ -108,38 +108,43 @@ function command_up {
 
 	if [ -d /opt/liferay/passwords ]
 	then
-		for service in $(ls /opt/liferay/passwords)
+		for service in /opt/liferay/passwords/*
 		do
+			[[ -e ${service} ]] || break
```

Comment:
We don't use this pattern, I guess its needed to check if there are files in the directory. Let's use find in the for loop instead.

Reply:
I changed it to a while loop based solution.